js/markdown/shared/Markdown.svelte

```diff
@@ -30,6 +31,7 @@
 	data-testid="markdown"
 	dir={rtl ? "rtl" : "ltr"}
 	use:copy
+	style="max-height: {height}; overflow-y: auto;"
```

Comment:
I feel like this style should only be applied if the height is not null

Reply:
I have made suggested changes and would request you to review and share your thoughts @freddyaboulton !